lib/tools/example_script.py

```diff
@@ -19,9 +19,9 @@
 Macros:
 API_KEY       -- an Admin API key. It can be found on the admin dashboard page.
 ROOT_URL      -- the root domain of the MarkUs instance.
-ROOT_DIR      -- the directory containing the group repos.
+ROOT_DIR      -- the directory containing the submission repos.
```

Comment:
@adisandro do you have time to revert changes to this file and markusapi.py? The only change I want to make to the existing scripts at this point is adding the cookie part. Anything else requires a bit more thought.